uaclient/defaults.py

```diff
@@ -20,6 +20,7 @@
 DOCUMENTATION_URL = (
     "https://discourse.ubuntu.com/t/ubuntu-advantage-client/21788"
 )
+GCP_SERVICE_ACCT_DOC_URL = "https://cloud.google.com/iam/docs/service-accounts"
```

Comment:
I think this should just be hardcoded in the message string. It's not a default value to any option the cli has.

Reply:
Yeah, we will extract it to the defaults when it used in more than one places.